Apps/W1/Shopify/app/src/Base/Pages/ShpfyShopCard.Page.al

```diff
@@ -974,6 +997,26 @@ page 30101 "Shpfy Shop Card"
                         Report.Run(Report::"Shpfy Sync Shipm. to Shopify");
                     end;
                 }
+                action(SyncPostedSalesInvoices)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Sync Posted Sales Invoices';
+                    Image = Export;
+                    Enabled = Rec."Posted Invoice Sync";
+                    Promoted = true;
+                    PromotedCategory = Category5;
+                    PromotedIsBig = true;
+                    PromotedOnly = true;
+                    ToolTip = 'Synchronize posted sales invoices to Shopify. The action can be used only if the Posted Invoice Sync field is enabled in the Shopify shop.';
+
+                    trigger OnAction();
+                    var
+                        ExportInvoicetoShpfy: Report "Shpfy Export Invoice to Shpfy";
+                    begin
+                        ExportInvoicetoShpfy.SetShop(Rec.Code);
+                        ExportInvoicetoShpfy.Run();
```

Comment:
I'd check if this has to be implemented using the Background Sync pattern they're using for importing orders?
It would make sense, as there could be a lot of invoices to sync...

Reply:
I wrote an email to Andrei regarding this.